core/cmd/shell_local_test.go

```diff
@@ -480,7 +480,6 @@ func TestShell_RebroadcastTransactions_AddressCheck(t *testing.T) {
 			}
 
 			set := flag.NewFlagSet("test", 0)
-			set.Set("evmChainID", testutils.SimulatedChainID.String())
```

Comment:
go 1.21 trips over this, and I think it was just revived from a stale PR anyways.